src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/services/ccd/CcdEventErrorHandler.java

```diff
@@ -26,38 +26,40 @@ public CcdEventErrorHandler(DeadLetterService deadLetterService,
     public void handleJsonError(ServiceBusReceiverClient receiver,
                                 ServiceBusReceivedMessage message,
                                 JsonProcessingException ex) {
-        log.error("Unable to parse incoming message with id '{}'", message.getMessageId(), ex);
+        String incomingMessage = new String(message.getBody().toBytes());
+        log.error("Unable to parse incoming message '{}'", incomingMessage, ex);
         String messageData = new String(message.getBody().toBytes());
 
         receiver.deadLetter(message, deadLetterService.handleParsingError(messageData, ex.getMessage()));
 
-        log.warn("Message with id '{}' was dead lettered", message.getMessageId(), ex);
+        log.warn("Message '{}' was dead lettered", incomingMessage, ex);
```

Comment:
Remove logging of entire message this might contain sensitive date use only the id